src/engine/renderer/gl_shader.cpp

```diff
@@ -546,6 +546,11 @@ static std::string GenEngineConstants() {
 			AddDefine( str, "r_ShowParallelShadowSplits", 1 );
 	}
 
+	if ( r_dynamicLight->integer )
+	{
+		AddDefine( str, "r_dynamicLight", r_dynamicLight->integer );
```

Comment:
`r_dynamicLight` is not set as `CVAR_LATCH` so you can't use it at compile time

Reply:
We may make all of them `CVAR_LATCH` then…

Reply:
`CVAR_LATCH` is now set.